### ISSUE TITLE
Control Sonos devices outside the iobroker LAN 

### ISSUE THREAD

**Issue by ymurawski** [ioBroker/ioBroker.sonos#189]
Hi, 
i have some different VLANS at home for different usages.
The iobroker is running at 192.168.1.200 and all my Sonos devices are in my IOT network under 192.168.10.x
i can access everything (ping) but when i enter the sonos devices in the adapter it just creates the devices but cant read any settings from them.

any advice ?

**Comment by Apollon77** [comment]
Are UDP/MDNS packages routet between the VLANs?

**Comment by ymurawski** [comment]
Yes its activated in all Networks. Devices like my shelly plugs wich are connected to the same IOT network just working fine.

**Comment by ymurawski** [comment]
> Are UDP/MDNS packages routet between the VLANs?


Yes its activated in all Networks. Devices like my shelly plugs wich are connected to the same IOT network just working fine.

**Comment by ymurawski** [comment]
Any news ? :(

**Comment by Apollon77** [comment]
No idea what you expect ... maybe also try asking in the Forum if other users have experiences with such setups? 

Me as a developer (if you would have Sonos at all) most likely would not have such a setup ... When I google that then I find several users in general have issues with such setups and works for others ... (not directly iobroker but in general): https://community.ui.com/questions/VLAN-inter-accessibility-for-MultiCast-devices-SONOS-Chromecast-Airtame-etc/f4d7a07c-d4ea-4238-8265-38ccd6e904c7 ... So I assume you need to investigate your self a bit more also.

Sorry that I can not provide a better solution for now

**Comment by RoccoNaut** [comment]
any news on this?
I either can listen to music when my Sonos are connected to the WiFi Network or  can use it for ioBroker when they are connected to the ioBroker VLAN. Both at the same time doesn't work.
